cts/Makefile.am

```diff
@@ -85,10 +76,12 @@ CLEANFILES	= $(builddir)/.regression.failed.diff
 clean-local:
 	rm -f scheduler/*/*.pe
 
-SUBDIRS	= benchmark scheduler
+SUBDIRS	= benchmark scheduler support
 
-cts-support-install: cts-support
-	./cts-support install
+cts-support-install:
+	make -C support cts-support
```

Comment:
I've been doing a lot of stuff with make at home recently, and you should apparently always use $(MAKE) -C for things like this (and the next target down).

Reply:
Oh, right, $(MAKE) $(AM_MAKEFLAGS) actually, I should have remembered that as I once made sure everything was using it :)

Reply:
Fixed (and rebased on current master)